lms/envs/production.py

```diff
@@ -1155,6 +1155,9 @@
 COLARAZ_NOTIFICATIONS = ENV_TOKENS.get(
     'COLARAZ_NOTIFICATIONS'
 )
+COLARAZ_JOB_ALERTS = ENV_TOKENS.get(
+    'COLARAZ_JOB_ALERTS'
+)
```

Comment:
If `COLARAZ_JOB_ALERTS` is not configured, the system will crash. Is this the desired behavior?
or can we avoid that and using a default value, simply disable the feature if this is not configured?

Reply:
How can you say it will break the system? have I missed something?

Reply:
I think it will not crash the system
![image](https://user-images.githubusercontent.com/42294172/83153934-f29ec580-a118-11ea-8b1e-678d5b47c02f.png)

Reply:
got it.